addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -33,17 +37,40 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
   loadOptions() {
     const metaGraph = this.args.graphs.metaGraph;
     const fieldOptions = this.args.field.options;
+    let conceptScheme = null;
 
     if (!hasValidFieldOptions(this.args.field, ['conceptScheme'])) {
-      return;
-    }
+      const scheme = this.args.formStore.any(
```

Comment:
if we are going this route, i would switch the order of the checks, first checking for the predicate version as that will become the default and the old one will be deprecated.

Reply:
updated it so it first checks the predicate and than the form:options if nothing was found for the predicate. Good one 👌🏻